cmd/kube-proxy/app/options.go

```diff
@@ -214,17 +211,22 @@ func (o *Options) Complete(fs *pflag.FlagSet) error {
 		// command line flags have priority). Otherwise `--config
 		// ... -v=5` doesn't work (config resets verbosity even
 		// when it contains no logging settings).
-		copyLogsFromFlags(fs, &c.Logging)
+		if err = copyLogsFromFlags(fs, &c.Logging); err != nil {
+			return err
+		}
```

Comment:
(a separate commit to fix the lint problems caused by moving the code wouldn't be terrible...)